cinema/views.py

```diff
@@ -1,11 +1,13 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets, mixins
+from rest_framework import viewsets, mixins, status
```

Comment:
Code Style: Consider grouping imports using parentheses for better readability, as suggested in the checklist.